addons/point_of_sale/tools/posbox/overwrite_before_init/etc/init_posbox_image.sh

```diff
@@ -34,23 +38,19 @@ rm -rf /usr/share/doc
 # python-usb in wheezy is too old
 # the latest pyusb from pip does not work either, usb.core.find() never returns
 # this may be fixed with libusb>2:1.0.11-1, but that's the most recent one in raspbian
-# so we install the latest pyusb that works with this libusb
-pip install pyusb==1.0.0b1
-pip install qrcode
-pip install evdev
-pip install simplejson
-pip install unittest2
+# so we install the latest pyusb that works with this libusb.
+# Even in stretch, we had an error with langid (but worked otherwise)
+pip3 install pyusb==1.0.0b1
+pip3 install evdev
 
 # --upgrade because websocket_client in wheezy is bad:
 # https://github.com/docker/compose/issues/1288
-pip install --upgrade websocket_client
+#pip3 install --upgrade websocket_client
```

Comment:
why? either remove or fix the command

Reply:
Because it was done because it was bad in wheezy and so it works without in stretch.  I could remove it, but you never know...